samples/word/accept_all_revisions/cs/Program.cs

```diff
@@ -66,7 +67,7 @@ static void AcceptAllRevisions(string fileName, string authorName)
                 }
                 else
                 {
-                    OpenXmlElement? nextSibling = insertion.NextSibling() ?? throw new System.NullReferenceException("NextSibling is null.");
+                    OpenXmlElement? nextSibling = insertion.NextSibling() ?? throw new ArgumentNullException("NextSibling is null.");
```

Comment:
nextSibling will be non nullable here

Reply:
`OpenXmlElement.NextSibling()` returns `OpenXmlElement?`